ckanext/dgu/theme/templates/package/new_and_edit.html

```diff
@@ -0,0 +1,18 @@
+{% extends "page.html" %}
+
+{% block optional_head %}
+  <script type="text/javascript">
+    var form_errors = {{ c.errors_json|safe }};
+    var publishers = {{ c.publishers_json|safe }};
+    window.DGU_RESOURCE_FORMATS = {{h.get_resource_formats()|safe}};
+  </script>
+  <script type="text/javascript" src="/scripts/dgu-package-form.min.js"></script>
+  <link rel="stylesheet" type="text/css" href="/css/jquery-ui-1.9.2.custom.datepicker.min.css" />
+  <script type="text/javascript" src="/scripts/vendor/jquery-ui-1.9.2.custom.datepicker.min.js"></script>
+{% endblock %}
+
+{% block primary_content_inner %}
+  {#- NB c is passed in, but deprecated #}
+  {{- h.snippet(form_snippet, c=c, **form_vars) -}}
```

Comment:
It comes from this new bit in the controller, copied from ckan master: https://github.com/datagovuk/ckan/pull/5/files#diff-d61fa02346248a2c5eed362244515307R557
Sorry, I should have mentioned this PR relies on this other PR: https://github.com/datagovuk/ckan/pull/5